cmd/determinize-prow-jobs/main.go

```diff
@@ -34,10 +34,20 @@ func bindOptions(flag *flag.FlagSet) *options {
 
 func determinizeJobs(prowJobConfigDir string) error {
 	errChan := make(chan error)
+
+	var errs []error
```

Comment:
The previous positioning of this created a deadlock, because the channel is unbuffered and we only started reading from it (thus freeing space) after we walked the filepath, but that doesn't continue when we are blocked writing to the channel in the syncrhonous part